instruqt-tracks/vault-advanced-data-protection-with-tokenization/track.yml

```diff
@@ -0,0 +1,376 @@
+slug: vault-advanced-data-protection-with-tokenization
+id: pm1e6okdgbx3
+version: 0.0.1
+type: track
+title: Vault ADP with Tokenization
+teaser: Leverage Vault's Transform Secret Engine (Tokenization) for protecting customer
+  PII data.
+description: |-
+  The Vault Transform Secret Engine provides the ability to tokenize sensitive data for storing outside of Vault.
+
+  When encrypting sensitive data, preservation of the original data format or length may be required to meet certain industry standards such as HIPAA or PCI.
+  To fulfill this requirement, the transform secrets engine can perform format preserving encryption (FPE).
+
+  However, there are organizations that care more about the irreversibility of the tokenized data and not so much about preserving the original data format.
+  Therefore, the transform secrets engine's FPE transformation may not meet the governance, risk and compliance (GRC) strategy they are looking for due to the use of reversible cryptography to perform FPE.
+
+  In this lab, you will use the Transform secret engine with an example Golang application and external Postgres database.
+icon: https://storage.googleapis.com/instruqt-frontend/assets/hashicorp/tracks/vault.png
+tags:
+- vault
+- adp
+- tokenization
+owner: hashicorp
+developers:
+- aklaas@hashicorp.com
+private: false
+published: true
+challenges:
+- slug: enable-the-transform-secret-engine
+  id: lz6iiyy29zf0
+  type: challenge
+  title: Enable the Transform (Tokenization) secret engine
+  teaser: In this first challenge, we will enable and configure Vault's Transform
+    secret engine with data tokenization.
+  notes:
+  - type: text
+    contents: |-
+      Secrets engines are Vault plugins that store, generate, tokenize, or encrypt data.
+      The Transform secrets engine functions as Vault's Encryption-as-a-Service and Tokenization solution.
+      The secret engine supports tokenization, format preserving encryption, and data masking transformations.
+      To learn more, see https://www.vaultproject.io/docs/secrets/transform/tokenization.
+
+      This lab usually takes 3-5 minutes to spin up.
+
+      ![Tokenization](https://github.com/Andrew-Klaas/instruqt-tokenization-lab/raw/main/assets/images/tokenization_overview.png)
```

Comment:
Please get this from inside this repo instead of from your personal repo.

Reply:
Fixed